packages/router/src/util.ts

```diff
@@ -8,22 +8,40 @@ const createNamedContext = <T>(name: string, defaultValue?: T) => {
 }
 
 /**
- * Get param name and type transform for a route
+ * Get param name, type, and slug for a route.
```

Comment:
You're the master of naming things. But personally I wouldn't call the match a "slug"

![image](https://user-images.githubusercontent.com/30793/140438900-009917bc-d818-4f3f-9f98-1045a6d6356d.png)

Reply:
Fair point, I suppose it can just be called `match`. I will make the change.